tests/test_io_utils.py

```diff
@@ -14,35 +15,67 @@
     "filename, mean, shape",
     [
         ("img.tif", 13577.494856, (2048, 2048, 3)),
-        ("img.jpg", 52.412425, (2048, 2048, 3)),
         ("img.png", 52.33736, (2048, 2048, 3)),
         ("img-8bit.jp2", 47.09060, (343, 343, 3)),
         ("img-15bit.jp2", 0.3041897, (1830, 1830)),
         ("img-16bit.jp2", 0.3041897, (1830, 1830)),
     ],
 )
 def test_img_read(input_folder: str, output_folder: str, filename: str, mean: float, shape: Tuple[int, ...]) -> None:
-    is_jpeg = filename.endswith("jpg")
     img = read_data(os.path.join(input_folder, filename))
 
+    if filename.endswith("txt"):
+        img = np.fromstring(img)
```

Comment:
there is no test case where the filename ends with `.txt`
also no reason for the data to be an image in that case, just use a human readable string. Good luck finding the difference between two 40.000 character strings (where one symbol changed due to encoding)

Reply:
removed